ChangeLog.md

```diff
@@ -1,3 +1,18 @@
+### 0.4.0
+
+#### chruby.sh
+
+* Use the installed Ruby directory name for setting `GEM_HOME`. (@eregon)  
+  This guarantees a unique `GEM_HOME` per installed Ruby, even if they have
+  the same `RUBY_VERSION`.  This is important for native extensions, which
+  might compile differently based on build-time flags such as `--enable-shared`.
+  This also fixes the bug that the `GEM_HOME` for non-MRI Ruby implementations
+  was shared even for different releases.
+
+  In practice, updating `chruby` means you will need to install gems again as
```

Comment:
This is a pretty disruptive change. I don't feel safe about springing it on users in version 0.4.0. This should be scheduled for a 1.0.0 release.

Reply:
Agreed, 1.0.0 would make sense for this.